packages/front-end/services/UserContext.tsx

```diff
@@ -232,9 +233,25 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
     if (!isAuthenticated) {
       return;
     }
-    updateUser();
+    void updateUser();
   }, [isAuthenticated, updateUser]);
 
+  // Refresh user and org after loading license
+  useEffect(() => {
+    if (orgId) {
+      void refreshOrganization();
+    }
+    if (isAuthenticated) {
+      void updateUser();
+    }
```

Comment:
I wonder if doing these two operations would be better suited at the 'success' stage of setting a license key (where we say `'License key accepted. Thank you!'`). Doing them in a `useEffect` has the slight risk of re-running when any of the dependencies change

Reply:
The problem is it needs to trigger on app bootup, or after a user logs in to an org, after the mongo record has been read